experiment/autobumper/bumper/bumper.go

```diff
@@ -261,3 +534,37 @@ func generatePRBody(images map[string]string, assignment string) string {
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
 	return prowSummary + "\n\n" + testImagesSummary + "\n\n" + assignment + "\n"
 }
+
+func getAssignment(oncallAddress string) string {
+	if oncallAddress == "" {
+		return ""
+	}
+	oncaller, err := getOncaller(oncallAddress)
+	if err == nil {
```

Comment:
Please indent the error flow

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Done.
Since the `err` needs to be used out of the `if` block, I will have to define the two vars first:
```
var oncaller string
var err error
```